core/src/utils/test/playwright/locator/utils/wait-for-custom-event.ts

```diff
@@ -0,0 +1,27 @@
+import type { Locator } from '@playwright/test';
+
+import type { E2ELocator } from '../../playwright-declarations';
+
+/**
+ * Waits for a custom event to be dispatched on an element handle. Fails after 5000ms without an event emission.
+ */
+export const waitForCustomEvent = (
```

Comment:
Is this still needed now that event spies exist? The descriptions in the `IonicPage` interface seem to indicate they perform the same functions.

Reply:
Nope, good catch! I can remove that & clean-up the page interface. Thinking it might be useful if we throw a quick `testing.md` in our playwright folder, for how to use our basic test utilities. Would be a decent reference for us as we leverage those custom function more & a great resource for future contributors.